### PR TITLE
make config better

### PR DESCRIPTION
**Pull request by wgyt**
solves #4
now config is:
```json
{ "usernamee":"wgyt", "projectid": "513842800", "commentid":"200363559"}
```
this is done in 1host.config.js

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 "use strict";
 
-module.exports = async (req, res) => {
+module.exports = async (req, res, config) => {
 	const fetch = require("node-fetch");
 	const db = new (require("@replit/database"))();
 	const retronid = require("retronid");
 	const ejs = require("ejs");
-	require("dotenv").config();
```
Comment (RedGuy12) [inline index.js:8]: don't remove this. this loads env variables from `process.env`. yes, replit does it automatically but other hosts don't.

Comment (RedGuy12) [reply index.js:8]: oh i forgot you aren't using env anymore

Comment (wgyt) [reply index.js:8]: this isn't needed because the config isn't loaded from env anymore, it uses 1host's config for modules

**Comment by wgyt** [comment]
Merging this pr

**Pull request merged**